go/src/koding/kites/kloud/stack/team.go

```diff
@@ -39,3 +42,35 @@ func (k *Kloud) TeamList(r *kite.Request) (interface{}, error) {
 
 	return TeamListResponse{Teams: teams}, nil
 }
+
+// WhoamiResponse represents a response value for a "team.whoami" kite method.
+type WhoamiResponse struct {
+	Team *team.Team `json:"team"`
+}
+
+// TestWhoami is a kite handler for a "team.whoami" kite method.
+func (k *Kloud) TeamWhoami(r *kite.Request) (interface{}, error) {
+	opts := &modelhelper.LookupGroupOptions{
+		Username:    r.Username,
+		KiteID:      r.Client.ID,
+		ClientURL:   r.Client.URL,
+		Environment: r.Client.Environment,
+	}
+
+	group, err := modelhelper.LookupGroup(opts)
+	if err != nil {
+		return nil, models.ResError(err, "jGroup")
+	}
+
+	members, _ := group.Counts["members"].(int)
```

Comment:
Done.

@ppknap This should also be dropped from `machine.list`. In separate PR. Does anything depends on it?